### PR TITLE
Working editing mode

### PR DESCRIPTION
**Pull request by fecso8787**
The editing mode enabled with the latest Leopoly branch's states.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
An admin must verify this patch for the pull request build to fire. Type "ok to test" to begin testing this pull request or "add to whitelist" to test this and all future pull requests from this collaborator.

**Comment by ZappoMan** [comment]
add to whitelist

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -2129,6 +2131,69 @@ void Application::paintGL() {
...
+        EntityItemID entityUnderSculptID;
+        entityUnderSculptID.data1 = LeoPolyPlugin::Instance().CurrentlyUnderEdit.data1;
+        entityUnderSculptID.data2 = LeoPolyPlugin::Instance().CurrentlyUnderEdit.data2;
+        entityUnderSculptID.data3 = LeoPolyPlugin::Instance().CurrentlyUnderEdit.data3;
+        for (int i = 0; i < 8; i++)
+            entityUnderSculptID.data4[i] = LeoPolyPlugin::Instance().CurrentlyUnderEdit.data4[i];
+
+        
+        static bool loool = false;
+        //if (!looool)
```
Comment (thoys) [inline interface/src/Application.cpp]: lol

**Comment by hifi-gustavo** [comment]
Build console output for commit: c18781e40b0861f87102724d6d524a95ea1ef66b available at: https://gist.github.com/67222d82433781bee9e320c89d6f9785<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9067-c18781e.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9067-c18781e.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 18e08ca344c7250b6812042fc8f1e351ccbc6dee available at: https://gist.github.com/67222d82433781bee9e320c89d6f9785<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9067-18e08ca.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9067-18e08ca.dmg

**Pull request closed without merging**